content/api/sending-domains.apib

```diff
@@ -410,346 +436,64 @@ Delete an existing sending domain.
-|------------------------|:-:       |---------------------------------------|
-|scph1015._domainkey.mail.example.com | TXT | v=DKIM1; k=rsa; h=sha256; p=MIGfMA0GCSqGSIb3DQEBAQUAA5GNADCBiQKBgQCzMTqqPX9jry+nKZjqYhKt5CP4+vBoEpf24POjc5ubWJQnZmY0wdBXawskxC7mBekUlAjOcsbZIhnFt+2asb1XTyLcTjGyqMvVcoUou6olzfMnfB06W9awRahQrrs9E0LZ4hYKSBDTm3MvoJo004+dNpTSnTlGqMyOoBuiD6KX8QIDAQAB |
-
-**CNAME** verification requires the following:
-  * <strong>SparkPost</strong> A valid CNAME record in DNS with value `sparkpostmail.com`
-  * <strong>SparkPost EU</strong> A valid CNAME record in DNS with value `eu.sparkpostmail.com`
-  * <span class="label label-warning"><strong>Enterprise</strong></span> A valid CNAME record in DNS with value `<public_tenant_id>.mail.e.sparkpost.com`
-
-An example CNAME record for a <strong>SparkPost</strong> customer with domain *mail<span></span>.example.com*:
-
-| Hostname         | Type     | Value                           |
-|------------------------|:-:       |---------------------------------------|
-|mail<span></span>.example.com | CNAME | sparkpostmail<span></span>.com |
-
-An example CNAME record for a <strong>SparkPost Enterprise</strong> customer with public_tenant_id *foo* and with domain *mail<span></span>.example.com*:
-
-| Hostname         | Type     | Value                           |
-|------------------------|:-:       |---------------------------------------|
-|mail<span></span>.example.com | CNAME | foo.mail.e.sparkpost<span></span>.com |
-
-**MX** verification is available to <strong>Enterprise</strong> customers only. There is no way to initiate MX verification through the /verify endpoint. Please contact your TAM if you want to verify your domain with MX.
+### List all Sending Domains [GET /sending-domains{?ownership_verified,dkim_status,cname_status,mx_status,abuse_at_status,postmaster_at_status,compliance_status,is_default_bounce_domain}]
 
-#### Using a Sending Domain as a Bounce Domain
-A Sending Domain is eligible to be used as a Bounce Domain if one of the following conditions is met:
-* A CNAME record in place and verified via `"cname_verify":true`
-* An MX verified domain (<span class="label label-warning"><strong>Enterprise</strong></span> only)
-
- Eligible domains may be used as a bounce domain by including it as part of the transmission return_path or SMTP MAIL FROM email address. Bounce domains are used to report bounces, which are emails that were rejected from the recipient server. By adding a bounce domain to your account, you can customize the address that is used for the `Return-Path` header, which is the destination for out of band (OOB) bounces.  For additional details on CNAME-verification, please see this [support article](https://www.sparkpost.com/docs/tech-resources/custom-bounce-domain/).
-
-The domain's `status` object is returned on success.
+List an overview of all sending domains. By default, all domains are returned. Use the query parameters to filter on the various status options. 
```

Comment:
"Return an array with all the sending domains in your account. Use the query parameters to filter on the various status options."